conf/configure.fv3.hera.intel

```diff
@@ -180,6 +180,6 @@ LDFLAGS += $(LIBS)
 
 ifdef InNemsMakefile
 FFLAGS += $(ESMF_INC)
-CPPFLAGS += -traditional
-EXTLIBS = $(NCEPLIBS) $(ESMF_LIB) $(LDFLAGS) $(NETCDF_LIB)
+CPPFLAGS += -DFV3_CPLD -traditional
```

Comment:
May I ask why do we need to add FV3_CPLD back?

Reply:
It will be removed.